f600233d1b154d507b782f84604bcb12/update/sys_script_796665eb871281108ea6cbb9cebb35f8.xml

```diff
@@ -15,7 +15,7 @@
         <condition/>
         <description/>
         <execute_function>false</execute_function>
-        <filter_condition table="x_g_dis_atat_organization">agency=19ba0874db9a5950b1227ea5f39619bf^organization_nameISNOTEMPTY^EQ<item display_table="x_g_dis_atat_agency" display_value="Defense Information Systems Agency (DISA)" endquery="false" field="agency" goto="false" newquery="false" operator="=" or="false" value="19ba0874db9a5950b1227ea5f39619bf"/>
+        <filter_condition table="x_g_dis_atat_organization">agency=fdd5245e875a9910bc86b889cebb35e3^organization_nameISNOTEMPTY^EQ<item display_table="x_g_dis_atat_agency" display_value="Defense Information Systems Agency (DISA)" endquery="false" field="agency" goto="false" newquery="false" operator="=" or="false" value="fdd5245e875a9910bc86b889cebb35e3"/>
```

Comment:
Simply updated referenced record in the BR.

Reply:
Example 
![OrganizationBR-bug_AT-8359](https://user-images.githubusercontent.com/89867104/203112008-22370b71-782d-4dd0-9013-53bdba035c55.png)